scripts/ci_python.js

```diff
@@ -39,7 +39,17 @@ try {
     let build_cmd = `${python} -m pip install -r requirements-dev.txt &&\
         ${python} setup.py build &&\
         ${python} -m flake8 perspective && echo OK &&\
-        ${python} -m pytest -v perspective --cov=perspective`;
+        ${python} -m pytest -v perspective --cov=perspective &&\
+        codecov --token 0f25973b-091f-42fe-a469-95d1c6f7a957`;
+    if (!PY2) {
+        // dont build docs/install check for python2
+        build_cmd += `&&\
+        make -C ./docs html &&\
+        ${python} -m pip install -U pip &&\
+        ${python} -m pip install . &&\
+        ${python} setup.py sdist &&\
+        ${python} -m pip install -U ./dist/perspective*`;
+    }
```

Comment:
`ci_python` and `build` (with a proper [`.perspectiverc`](https://perspective.finos.org/docs/md/development.html#building)) are identical, except the former runs `install` and some other sdist tasks now.  Can we
1. Merge this with `build` so our CI and dev builds are the same?
2. Remove 'pip install` Install the necessary python modules as part of the docker image, so this is consistent, build time is conserved, etc?
3. Move the sdist tasks to `test` or at least put it behind a build flag so it is not mandatory?

Reply:
1 sure
2 this makes dependency adding/changing more annoying, it will require a docker image upgrade anytime we add or modify a dependency. Doesn't matter to me, you'll be trading faster incremental builds for much longer occasional builds (when you have to rebuild and repush the docker image).
3 I don't care where they go so long as they are tested